django_elasticsearch_dsl/fields.py

```diff
@@ -5,7 +5,7 @@
 from django.db import models
 from django.db.models.fields.files import FieldFile
 from django.utils.encoding import force_text
-from django.utils.functional import Promise
+from django.utils.functional import Promise, SimpleLazyObject
```

Comment:
```suggestion
from django.utils.functional import Promise
```
One small nit, need to remove importing SimpleLazyObject